### DISCUSSION TITLE
Looks great! Using it as digital signage

### DISCUSSION THREAD

**Discussion by andiohn** [debloper/piosk#46] [Show and tell]
Hey all,
This simple tool makes for excellent signage as it is! Currently it's displaying 6 sites, and the only real change I made to it so far is to have it reboot once a day. I'd love to 'blank' the display on a schedule as well, but it's working great. 

After configured, I turned on the overlay file system and mounted the boot as read only. The websites it loads are redirected from another web server, so to update them, I just change the redirect and it will get it on the next boot. 

Very happy, and hopefully because there's not much to go wrong, it will just work and work and work. 

I have a lot of experience with digital signage and so far, this is the best simple option I've found. I'm excited to see this improve and improve!

![PXL_20241009_152623006](https://github.com/user-attachments/assets/3343c2d2-51a8-4296-931e-044567dc0b46)

**Comment by andiohn** [comment]
With 2 small improvements, it would be very slick. If we could:
1. Display a 'loading' or something as it's loading the pages and stays that way until all of them are loaded
2. Schedule blanking the display (either just show black or sleep or turn off the display) and schedule rebooting from the web

If we can just get solutions to these 2 things, it looks really professional.

**Comment by andiohn** [comment]
Just to further make things interesting, I'm going to try with other architectures and we'll see how we're doing! Like N100 etc. 

https://www.jeffgeerling.com/blog/2024/radxa-x4-sbc-unites-intel-n100-and-raspberry-pi-rp2040

Because technically, if it's just running raspi os, we might be good for more other computers because I love how simple this setup is.

**Comment by debloper** [comment]
Whoa, this is awesome... thank you so much!

I'm super glad to see the real world issues you're surfacing, and finding hotfixes to make it work.

Also, I'm working with N100 as well (Radxa X4, LattePanda Mu etc), so I have the hardware to test/debug - please keep us posted on how it goes. From what I can imagine, there shouldn't be issues running it on a different architecture... the only real requirement is Wayland compositor with Wayfire WM.

For the feature-requests - I have them in my mind, will address them as soon as I get the time. Which is also why I'm keeping them sorted and updated as much possible, so I don't find myself in a mess when I finally find some time to fix them.

**Comment by espec** [comment]
How did you hide the cursor? This has been my challenge. I ended up manually moving it offscreen, but needs to be done with every reboot.

**Reply by tismofied** [reply]
The answer is at https://github.com/debloper/piosk/issues/23

**Reply by espec** [reply]
Well, that discussion makes me feel better ... so many ways and none seem right or straightforward. I'll follow your steps, thanks so much.
